app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,25 +1,29 @@
-- if @logs.blank? && @context != 'calendar'
-  = content_tag(:div, t('common.no_records'), class: 'no-records-msg')
+- @context = @context ? @context : 'logs_list'
```

Comment:
Usually the calendar context (which is both the timeline table and calendar) and the dashboard context are passed in as the context variable. However, this is not the case when the logs list is a page. So context is being set here.